src/NuGet.Protocol.Catalog/Models/CatalogPageContext.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Catalog
+{
+    public class CatalogPageContext
```

Comment:
Just curious, why are you adding this? Does anything need to read/write this?

Reply:
I added it so I can use the catalog models in tests (for easy serialization). This product code needs the context:
https://github.com/NuGet/NuGet.Services.Metadata/blob/e921a7eef360ee6bcc11b7ca9f94d96e93193a1c/src/Catalog/CatalogCommitItem.cs#L69-L74